mojo/services/keyboard/interfaces/BUILD.gn

```diff
@@ -9,4 +9,10 @@ mojom("interfaces") {
   sources = [
     "keyboard.mojom",
   ]
+
+  import_dirs = [ get_path_info("../../", "abspath") ]
+
+  deps = [
+    "../../native_viewport/interfaces",
```

Comment:
We usually write these as relative to source root. So something like "//mojo/services/native_viewport/interfaces" should work